manta-js/package/src/privateWallet.ts

```diff
@@ -306,22 +345,24 @@ export class MantaPrivateWallet implements IMantaPrivateWallet {
   }
 
   /// Private helper method for internal use to initialize the Polkadot.js API with web3Extension.
-  private static async initApi(env: Environment, network: Network, loggingEnabled:boolean): Promise<InitApiResult> {
+  private static initApi(env: Environment, network: Network, loggingEnabled: boolean): InitApiResult {
```

Comment:
Doesn't this need to continue being async and just not await it when you call it ?

Reply:
It doesn't need to be `async` because there's no `await` in the function body

Reply:
But if its not async wouldn't it block the execution and wait for the API to make a connection ?
Or does it return a nullptr API ?

Reply:
You can initialize the api class without a connection